lib/puppet/type/firewall.rb

```diff
@@ -1191,5 +1205,11 @@ def should_to_s(value)
       self.fail "Parameter 'stat_probability' requires 'stat_mode' to be set to 'random'"
     end
 
+    if value(:bridge)
```

Comment:
There is no property called `bridge` yet. Do you want to make one for the ipv6 provider?